### ISSUE TITLE
collate_fn function needs attention

### ISSUE THREAD

**Issue by joaocarvalho00** [joaocarvalho00/AutomatedMarineGrowth-Classification#4]
def collate_fn(self, batch):
        ims, masks = list(zip(*batch))
        ims = torch.cat([tfms(im.copy()/255.)[None] for im in ims]).float().to(device)
        ce_masks = torch.cat([torch.Tensor(mask[None]) for mask in masks]).long().to(device)
        return ims, ce_masks

device is not passed to class when instantiated, needs fix
